VariablesHomework/VariablesHomework.cs

```diff
@@ -148,9 +149,9 @@ public void Conversion()
         {
             int i = 13;
             double d = i;
-            Assert.IsTrue(d == /*Answer here*/); // Make this pass
+            Assert.IsTrue(d == (double)i ); // Make this pass
```

Comment:
I'm actually looking for a literal value here (e.g., `d == 97.5`, `d == 4`, `d == "Hello"`, etc.)

Reply:
Changed  to (d == 13);